qutebrowser/utils/log.py

```diff
@@ -526,6 +529,31 @@ def hide_qt_warning(pattern: str, logger: str = 'qt') -> typing.Iterator[None]:
         logger_obj.removeFilter(log_filter)
 
 
+def init_from_config(conf: 'configmodule.ConfigContainer') -> None:
+    """Initialize logging settings from the config.
+
+    init_log is called before the config module is initialized, so config-based
+    initialization cannot be performed there.
+
+    Args:
+        conf: The global ConfigContainer.
+              This is passed rather than accessed via the module to avoid a
+              cyclic import.
+    """
+    assert _args is not None
+    ramlevel = conf.logging.level.ram
+    consolelevel = conf.logging.level.console
+    if ram_handler and ramlevel:
+        init.info("Configuring RAM loglevel to %s", ramlevel)
+        ram_handler.setLevel(LOG_LEVELS[ramlevel])
```

Comment:
```suggestion
        ram_handler.setLevel(LOG_LEVELS[ramlevel].upper())
```

(with the change above)